File: fern/snippets/tool-calling/versioning/python/per-execution-override.py
from composio import Composio

composio = Composio(api_key="YOUR_API_KEY")

# Specify version directly in execute call
result = composio.tools.execute(
    "GITHUB_CREATE_ISSUE",
    {
        "user_id": "user-123",
        "arguments": {
            "repo": "my-repo",
            "title": "Critical fix"
        },
        "toolkit_version": "20250116_00"  # Version for this execution
    }
)

File: fern/snippets/tool-calling/versioning/typescript/per-execution-override.ts
import { Composio } from "@composio/core";

const composio = new Composio({ apiKey: "YOUR_API_KEY" });

// Specify version directly in execute call
const result = await composio.tools.execute({
    toolSlug: "GITHUB_CREATE_ISSUE",
    userId: "user-123",
    arguments: {
        repo: "my-repo",
        title: "Critical fix"
    },
    toolkitVersion: "20250116_00"  // Version for this execution
});